server/core/server.go

```diff
@@ -74,18 +76,24 @@ type AccountServer struct {
 
 // NewAccountServer creates a new account server with a default logger
 func NewAccountServer() *AccountServer {
-	return &AccountServer{
-		logger: logging.NewNATSLogger(logging.Config{
-			Colors: true,
-			Time:   true,
-			Debug:  true,
-			Trace:  true,
-		}),
+	ac := &AccountServer{
+		logger: NewNilLogger(),
+	}
+	return ac
+}
+
+func (server *AccountServer) ConfigureLogger() natsserver.Logger {
+	opts := server.config.Logging
+	if isWindowsService() {
```

Comment:
Based on previous comment, we may want to have a server's config setting that "disables" logging, or allow to set a custom logger that would be set to the NilLogger.